assets/pages/new-contributions-landing/components/SetPassword/SetPasswordForm.jsx

```diff
@@ -125,10 +125,10 @@ function SetPasswordForm(props: PropTypes) {
           autoComplete="off"
           value={props.password}
           onInput={props.updatePassword}
-          pattern={'^.{6,72}$'}
+          pattern={'^.{6,20}$'}
           isValid={props.password.length >= 6 && props.password.length <= 72}
```

Comment:
oops, you need to change this line too!

Reply:
`props.password.length <= 72`

Reply:
ah, thanks! I assumed `pattern` was used for the check